src/KomaMRI.jl

```diff
@@ -28,5 +28,6 @@ include("KomaUI.jl")
 # Export the UI and the observables
 export KomaUI
 export sys_ui, seq_ui, obj_ui, raw_ui, img_ui
+export enable_unsafe_electron
```

Comment:
You don't need to export it. Check PlotlyJS's PR: https://github.com/JuliaPlots/PlotlyJS.jl/pull/499